docker/Dockerfile.release

```diff
@@ -1,4 +1,11 @@
 FROM ubuntu:xenial
+RUN apt-get update && \
```

Comment:
None of the dockerfiles in our repos (proxy or pilot) have one - is it required ?